salt/minion_auth/srv/salt/_runners/foreman_https.py

```diff
@@ -19,7 +19,7 @@ def salt_config():
     """
     Read the foreman configuratoin from FOREMAN_CONFIG
     """
-    with open(FOREMAN_CONFIG, 'r') as config_file:
+    with open(FOREMAN_CONFIG) as config_file:
```

Comment:
I think "r" is default anyway, but isn't it better to keep it explicitly here?

Reply:
Was done by pyupgrade https://github.com/asottile/pyupgrade?tab=readme-ov-file#redundant-open-modes